.gitlab-ci.yml

```diff
@@ -292,3 +301,31 @@ publish-docs:
     - git push origin gh-pages --force
   after_script:
     - rm -rf .git/ ./*
+
+#### stage:                        fuzz
+
+fuzz-tests:
+    stage:                           fuzz
+    <<:                              *docker-env
+    variables:
+        # The QUICKCHECK_TESTS default is 100
+        QUICKCHECK_TESTS:            1000
+    rules:
+        - if: $CI_PIPELINE_SOURCE == "schedule"
+        - if: $CI_COMMIT_REF_NAME == "master"
+    script:
+        - cargo test --verbose --features ink-fuzz-tests --no-fail-fast --workspace -- fuzz_
```

Comment:
`--features is not allowed in the root of a virtual workspace`

Reply:
Argh, so do we have to iterate through our crates and test each one with this feature instead?

Reply:
seems like yeah

Reply:
```suggestion
        - cargo test --verbose --features ink-fuzz-tests --no-fail-fast --workspace -- fuzz_
        - for crate in ${ALL_CRATES}; do
            cargo test --verbose --features ink-fuzz-tests --manifest-path crates/${crate}/Cargo.toml --no-fail-fast --workspace -- fuzz_;
          done
```

Reply:
but then you will have 
```

error: Package `ink_storage_derive v3.0.0-rc2 (/builds/crates/storage/derive)` does not have these features: `ink-fuzz-tests`
error: Package `ink_allocator v3.0.0-rc2 (/builds/crates/allocator)` does not have these features: `ink-fuzz-tests`
error: Package `ink_prelude v3.0.0-rc2 (/builds/crates/prelude)` does not have these features: `ink-fuzz-tests`
error: Package `ink_primitives v3.0.0-rc2 (/builds/crates/primitives)` does not have these features: `ink-fuzz-tests`
error: Package `ink_lang v3.0.0-rc2 (/builds/crates/lang)` does not have these features: `ink-fuzz-tests`
error: Package `ink_lang_macro v3.0.0-rc2 (/builds/crates/lang/macro)` does not have these features: `ink-fuzz-tests`
error: Package `ink_lang_ir v3.0.0-rc2 (/builds/crates/lang/ir)` does not have these features: `ink-fuzz-tests`
```

Reply:
Okay, so seems like we have two options:
* Hardcode the packages which have this feature
* Introduce the feature for all packages, independent of if they have fuzz tests yet or not.

I lean towards the second one, since we then won't forget including a package if fuzz tests arrive for one. Wdyt?

Reply:
would be way more valid taking the second option and write a fuzz test for the each pkg ;)